client/basic-network/docker-compose.yml

```diff
@@ -82,6 +82,7 @@ services:
       - CORE_LEDGER_STATE_COUCHDBCONFIG_USERNAME=
       - CORE_LEDGER_STATE_COUCHDBCONFIG_PASSWORD=
       - CORE_CHAINCODE_MODE=${CORE_CHAINCODE_MODE:-net} # VSCODE
+      - CORE_PEER_NETWORKID=${COMPOSE_PROJECT_NAME} # VSCODE
```

Comment:
This change modifies the prefix of all chaincode containers and images to `fabricvscodelocalfabric` (or whatever the local Fabric is called) instead of the default, which is `dev`. By having a specific name, we can easily identify chaincode containers and images that a particular local Fabric has built - see below.